lld/ELF/SyntheticSections.cpp

```diff
@@ -801,7 +801,7 @@ uint64_t MipsGotSection::getGlobalDynOffset(const InputFile *f,
                                             const Symbol &s) const {
   const FileGot &g = gots[f->mipsGotIndex];
   Symbol *sym = const_cast<Symbol *>(&s);
-  return g.dynTlsSymbols.lookup(sym) * config->wordsize;
+  return g.dynTlsSymbols.lookup(sym) * (uint64_t)config->wordsize;
```

Comment:
1468948 [Unintentional integer overflow](https://scan6.scan.coverity.com/doc/en/cov_checker_ref.html#static_checker_OVERFLOW_BEFORE_WIDEN)
The expression's value may not be what the programmer intended, because the expression is evaluated using a narrow (i.e. few bits) integer type.

In lld::​elf::​GotSection::​getGlobalDynOffset(lld::​elf::​Symbol const &): An integer overflow occurs, with the result converted to a wider integer type ([CWE-190](http://cwe.mitre.org/data/definitions/190.html))